mux.go

```diff
@@ -363,6 +372,15 @@ func (r *Router) Walk(walkFn WalkFunc) error {
 	return r.walk(walkFn, []*Route{})
 }
 
+// CleanRedirectUse308 defines the redirect code following a path cleaning. The
+// default value is false - meaning it will use 301 (Moved Permanently) code.
```

Comment:
As a user, what would a 308 get me over a 301?

(Note: *I* know, but someone trying to learn mux will not)